processor/src/main/java/tw/ktrssreader/processor/util/TypeExtensions.kt

```diff
@@ -45,22 +45,35 @@ fun String.isPrimitive(): Boolean = primitiveJavaPaths.any { this.contains(other
 fun String.getFuncName(): String {
     return when {
         this.startsWith(GOOGLE_PREFIX) || this.startsWith(ITUNES_PREFIX) ->
-            "get${this.substringAfterLast(':').capitalize(Locale.ROOT)}"
-        else -> "get${this.capitalize(Locale.ROOT)}"
+            "get${this.substringAfterLast(':')
+                .replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.ROOT) else it.toString() }}"
```

Comment:
Nice, and how about we wrap the following line into a function or extension? Since it was used in this file frequently? 
```kotlin
replaceFirstChar { if (it.isLowerCase()) it.titlecase(Locale.ROOT) else it.toString() }}
```

Reply:
@ivanisidrowu should be fixed in latest commits let me know what you think. 

btw how can I test this locally? I see no `publishToMavenLocal` in `./gradlew tasks` :thinking:

Reply:
Sorry, we don't have the Gradle task.
However, you can test them by running the local and instrumental tests via the project.

Tests should be found under these directories:
`KtRssReader/android/src/androidTest`
`KtRssReader/android/src/test`
`KtRssReader/processorTest/src/androidTest`
`KtRssReader/processorTest/src/test`

Click the button in the editor to run the tests.
 
<img width="415" alt="Screen Shot 2021-05-07 at 7 06 12 PM" src="https://user-images.githubusercontent.com/6141401/117441160-8f8ff280-af67-11eb-89f2-1cd762bac3ee.png">